stdlib/filepath/src/test/java/org/ballerinalang/stdlib/filepath/nativeimpl/PathTest.java

```diff
@@ -399,8 +399,9 @@ public void testResolveNonExistencePath() {
         assertEquals(returns.length, 1);
         assertTrue(returns[0] instanceof BError);
         BError resolveError = (BError) returns[0];
-        assertEquals(resolveError.getReason(), Constants.FILE_NOT_FOUND_ERROR);
-        log.info("Ballerina error: " + resolveError.getDetails().stringValue());
+        assertEquals(resolveError.getMessage(),
+                     "File does not exist at src/test/resources/data-files/test_nonexist.txt");
```

Comment:
Same as above. shall we remove the file path and check only the first part

Reply:
Fixed in 6f0cd9f3156126d3a221e576cc91061d5656b530